x/feemarket/simulation/genesis.go

```diff
@@ -4,14 +4,16 @@ import (
 	"encoding/json"
 	"fmt"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 
 	"github.com/tharsis/ethermint/x/feemarket/types"
 )
 
 // RandomizedGenState generates a random GenesisState for nft
 func RandomizedGenState(simState *module.SimulationState) {
-	params := types.NewParams(simState.Rand.Uint32()%2 == 0, simState.Rand.Uint32(), simState.Rand.Uint32(), simState.Rand.Uint64(), simState.Rand.Int63())
+	params := types.NewParams(simState.Rand.Uint32()%2 == 0, simState.Rand.Uint32(), simState.Rand.Uint32(), simState.Rand.Uint64(), simState.Rand.Int63(), sdk.NewDecFromInt(sdk.NewInt(simState.Rand.Int63())))
```

Comment:
@fedekunze , regarding https://github.com/tharsis/ethermint/pull/1104#issuecomment-1141367359
> @loredanacirstea can you update the simulations to include the transaction fee?

I updated the simulation for the new param here. Is there anything else to update?

Reply:
`test-sim-<simulation>` CI tests are failing, possibly because every other module defines transactions with 0 fees in their simulation. Maybe we should change it to 0 to prevent the simulations from failing

Reply:
Done. Simulation tests are no longer failing.
The rpc tests are failing from the flakiness of `TestEth_Pending_GetTransactionByHash`. They need to be rerun, but I cannot trigger this.